docs/book/src/plugins/plugins.md

```diff
@@ -19,7 +19,7 @@ This section details how to extend Kubebuilder and create your plugins following
 <aside class="note">
 <h1>Note</h1>
 
-For further information check the design proposal docs [Extensible CLI and Scaffolding Plugins: phase 1][plugins-phase1-design-doc] and [Extensible CLI and Scaffolding Plugins: phase 1.5][plugins-phase1-design-doc-1.5]. 
+For further information check the design proposal docs [Extensible CLI and Scaffolding Plugins: phase 1][plugins-phase1-design-doc], [phase 1.5][plugins-phase1-design-doc-1.5] and [phase 2][plugins-phase2-design-doc] . 
```

Comment:
The phase 2 is not here because it is not implemented yet. It is the section vision for plugins see: https://book.kubebuilder.io/plugins/creating-plugins.html#future-vision-for-kubebuilder-plugins

Reply:
oh ohkay so while creating a new plugin, on will have to refer the phase 1 & 1.5?

Reply:
oh ohkay so while creating a new plugin, one will have to refer the phase 1 & 1.5?

Reply:
Yes, I think if you add phase 2 here it might lead to the idea that is implemented right? 

we can add something here as a note with a link to suggest to authors check what is its vision to the future

Reply:
okay.

Reply:
Hey @camilamacedo86, I made the changes can you take a look?